### DISCUSSION TITLE
Sponsorship?

### DISCUSSION THREAD

**Discussion by davidroman0O** [opentofu/opentofu#415] [Q&A] [answered]
I see that everyone is working hard on OpenTf, will we be able to sponsor to help financially? :)

**Comment by DicsyDel** [comment, marked as answer]
Hi @davidroman0O,

We're still ironing a lot of legal and organizational things. As soon as we legally can start accepting donations and have a financial management / planning in place, we will announce how to donate.

Regards,
Igor

**Reply by davidroman0O** [reply]
Perfect! Can't wait for it :)
